smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/XMLHttpResponseTraitPayload.kt

```diff
@@ -1,35 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-
-package software.amazon.smithy.swift.codegen.integration.httpResponse.bindingTraits
-
-import software.amazon.smithy.model.knowledge.HttpBinding
-import software.amazon.smithy.model.shapes.Shape
-import software.amazon.smithy.swift.codegen.SwiftWriter
-import software.amazon.smithy.swift.codegen.integration.HttpBindingDescriptor
-import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
-import software.amazon.smithy.swift.codegen.integration.httpResponse.HttpResponseBindingRenderable
-
-class XMLHttpResponseTraitPayload(
```

Comment:
`XMLHttpResponseTraitPayload` is renamed to `HTTPResponseTraitPayload`.  Not sure why the rename isn't tracked.